doc/classes/VisualShaderNodeFloatFunc.xml

```diff
@@ -1,81 +1,116 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualShaderNodeFloatFunc" inherits="VisualShaderNode" version="4.0">
 	<brief_description>
+		A scalar floating-point function to be used within the visual shader graph.
 	</brief_description>
 	<description>
+		Accept a floating-point scalar ([code]x[/code]) to the input port and transform it according to [member function].
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="function" type="int" setter="set_function" getter="get_function" enum="VisualShaderNodeFloatFunc.Function" default="13">
+			A function to be applied to the scalar. See [enum Function] for options.
 		</member>
 	</members>
 	<constants>
 		<constant name="FUNC_SIN" value="0" enum="Function">
+			Returns the sine of the parameter. Translates to [code]sin(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_COS" value="1" enum="Function">
+			Returns the cosine of the parameter. Translates to [code]cos(x)[/code] in Godot Shader Language.
 		</constant>
 		<constant name="FUNC_TAN" value="2" enum="Function">
+			Returns the tangent of the parameter. Translates to [code]tan(x)[/code] in Godot Shader Language.
```

Comment:
in _the_ Godot Shader Language

Same for below mentions of Godot Shader Language